jsonnet/thanos-querier.jsonnet

```diff
@@ -218,7 +218,12 @@ local authorizationRole =
                         cpu: '10m',
                       },
                     },
-                    ports+:: {},
+                    ports: [
+                      {
+                        containerPort: 9090,
+                        name: 'http',
+                      },
+                    ],
```

Comment:
Maybe we should add a TODO to remove this, because I think it's only needed until we configure the TLS from ruler to querier.

Reply:
I seen Simon remove it in his PR, think it was closed though, will do a in a follow up!